examples/tests/sdk_tests/api/delete.w

```diff
@@ -19,20 +20,22 @@ api.delete("/path", inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
   };
 });
 
+/// https://github.com/winglang/wing/issues/3342
+if (util.env("WING_TARGET") != "tf-aws") {
+  test "http.delete and http.fetch can preform a call to an api" {
+    let url = "${api.url}/path?all=true&page=6";
+    let response: http.Response = http.delete(url);
+    let fetchResponse: http.Response = http.fetch(url, {"method" => http_DELETE});
 
-test "http.delete and http.fetch can preform a call to an api" {
-  let url = "${api.url}/path?all=true&page=6";
-  let response: http.Response = http.delete(url);
-  let fetchResponse: http.Response = http.fetch(url, {"method" => http_DELETE});
+    // TODO: adding a fetch request when the enums 
```

Comment:
I know this comment is not new, but curious is this supposed to be `adding a fetch request with enums` or is the second half of the comment missing?

Reply:
a leftover probably, thanks!!!